src/bun.js/bindings/BunProcess.cpp

```diff
@@ -291,16 +291,32 @@ JSC_DEFINE_HOST_FUNCTION(Process_functionDlopen,
 
     RETURN_IF_EXCEPTION(scope, {});
 #if OS(WINDOWS)
-    CString utf8 = filename.utf8();
-    HMODULE handle = LoadLibraryA(utf8.data());
+    HMODULE handle;
+    printf("Trying to load %s\n", filename.utf8().data());
+    if (filename.is8Bit()) {
+        if (charactersAreAllASCII(filename.characters8(), filename.length())) {
+            CString utf8 = filename.utf8();
+            handle = LoadLibraryExA(utf8.data(), NULL, LOAD_WITH_ALTERED_SEARCH_PATH);
+        } else {
+            filename.convertTo16Bit();
+            handle = LoadLibraryExW((LPCWSTR)filename.characters16(), NULL, LOAD_WITH_ALTERED_SEARCH_PATH);
+        }
+    } else {
+        handle = LoadLibraryExW((LPCWSTR)filename.characters16(), NULL, LOAD_WITH_ALTERED_SEARCH_PATH);
```

Comment:
instead of loading all the symbols for this, maybe let's just choose one? we could always convert to 16 bit